### DISCUSSION TITLE
Define remote users to time management

### DISCUSSION THREAD

**Discussion by Nickyes** [AntonyCorbett/OnlyT#445] [Ideas]
Hello, thanks again for this tool

Is possible to create an admin group of users that, in addition to the PC client, can remotely edit, start and stop time?
If i can understand now the possibilty is given to ALL remote users, not to dedicated ones.
In this way will be possible to give only to someone this ability, which can operate remotely. 

As i mentioned above, this approved user could be also EDIT the time (as the PC client actually can do). In this way, directing would be relieved of this responsibility and everyone's duties would be better rebalanced.

Thanks a lot

**Comment by AntonyCorbett** [comment]
@Nickyes Thanks for your question. It's not clear what remote software you are using. 

There are several OnlyT remote apps available for iOS and Android. There is also a read-only remote web clock (see OnlyT Settings to configure)

Perhaps you could provide more details?

**Reply by Nickyes** [reply]
Yes i mean these two apps...
Asked to add some different user permission for brothers who use OnlyT Remote app, so that only a group (or maybe just one) can manage the time and not all brothers, to whom you only want to remotely show the timer (and don't edit or start)

Of course, that group or that designated brother would be in addition the full control that already has the main OnlyM client on PC

Thanks dear brother :)
